src/main/java/reposense/report/RepoCloner.java

```diff
@@ -79,32 +85,54 @@ private boolean isPreviousRepoDifferent() {
         return prevIndex == index || !configs[prevIndex].getLocation().equals(configs[index].getLocation());
     }
 
+    /**
+     * Spawns a process to clone repo specified in {@code repoConfig}. Does not wait for process to finish executing.
+     * Can only handle a maximum of one spawned process at any time and subsequent calls are ignored if current process
```

Comment:
This line can just simplify to `Should only handle a maximum of one spawned process at any time.`